### PR TITLE
update xla commit - check for exist pr

### PR DESCRIPTION
**Pull request by clee2000**
as in title, check for existing pr, if exists -> update that pr, if not exist -> make new pr

### CODE REVIEW

File: .github/workflows/update-xla-commit-hash.yml [hunk 1]
```diff
@@ -20,38 +20,59 @@ jobs:
         with:
           fetch-depth: 1
           submodules: false
+          token: ${{ secrets.PAT }}
```
Comment (janeyx99) [inline .github/workflows/update-xla-commit-hash.yml:23]: Did you already add this secret to our repo?

Comment (clee2000) [reply .github/workflows/update-xla-commit-hash.yml]: oops

File: .github/workflows/update-xla-commit-hash.yml [hunk 2]
```diff
@@ -20,38 +20,59 @@ jobs:
         with:
           fetch-depth: 1
           submodules: false
+          token: ${{ secrets.MERGEBOT_TOKEN }}
```
Comment (malfet) [inline .github/workflows/update-xla-commit-hash.yml:23]: Hmm, can we use a github token instead?
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

**Review by malfet — APPROVED** [review]
Looks good to me, but please elaborate why mergebot rather than standard github token needs to be used here?

File: .github/workflows/update-xla-commit-hash.yml (hunk 2 above)
Comment (clee2000) [reply .github/workflows/update-xla-commit-hash.yml:23]: if you push using the github token, it doesnt retrigger workflows
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow

**Pull request closed without merging**